src/commands/hugo-build.yml

```diff
@@ -16,7 +16,23 @@ parameters:
     description: "Additional flags to pass when running Hugo (e.g. -DF)."
     type: string
     default: ""
+  asciidoc:
```

Comment:
If you want to use the orb job, you'll still need to update `src/jobs/build.yml` to accept the param and add it to the step where this command is called: https://github.com/CircleCI-Public/hugo-orb/blob/master/src/jobs/build.yml#L46

Reply:
@dsayling Thanks for pointing this out. Resolved in commit 6266764a8f74efa9d4d9bed152405b42589bc93d.